apps/system/utils/README.md

```diff
@@ -333,6 +335,30 @@ Enable CONFIG_DEBUG_MM_HEAPINFO.
 Debug options -> Enable Debug Output Features to y
 ```
 
+### Heapinfo for User defined Group
+With specified task/thread name, heapinfo can show User defined group heap memory usage.
+
+Enable *CONFIG_HEAPINFO_GROUP* to use this option.
+```
+Application Configuration -> System Libraries and Add-Ons -> heapinfo -> Enable User defined Group Memory Usage -> set Thread Name List for measuring Alloc size
+```
+
+- Thread Name List for measuring Alloc size can be set like below:  
+  Task/Thread name is separated with ',' and Group is separated with '/'.  
+  For example, "abc,def/ghi,jklmn/opqr".
+
+```
+****************************************************************
+Heap Allocation Information per User defined Group
+****************************************************************
+ PEAK | HEAP_ON_PEAK | STACK | THREADS_IN_GROUP
```

Comment:
Why not ```STACK_ON_PEAK```? It could not be sum of stack of all threads.
On peak case, one of threads can be un-alive.

Reply:
done